Changed bio lab recipes

Peter5930 opened the pull request:
Added a dirt_to_coarse_dirt recipe (grass and mycelium won't grow on coarse dirt, so it's useful for landscaping for a bare-earth look, plus has it's own texture) and removed the make_mushroom_stem and make_mushroom_pores recipes since those blocks haven't been obtainable as items since MC 1.5 and the recipes both just produce the same brown mushroom cap blocks as the -make_mushroom_brown recipe.

Peter5930 committed:
Update config.yml

Added coarse dirt recipe to bio lab

Peter5930 committed:
Update config.yml

Removed mushroom stem and pore block recipes from the bio lab since it's not been possible to obtain those blocks as items since MC 1.5, so the recipes confusingly both just produce brown mushroom cap blocks with a durability of 0, identical to the - make_mushroom_brown recipe.  Blame Mojang; you can now only get the other mushroom blocks by growing mushrooms with bonemeal or using the /setblock command.

TealNerd commented on FactoryMod/config.yml:
```diff
@@ -1522,6 +1521,19 @@ recipes:
       mycelium:
         material: MYCEL
         amount: 64
+  dirt_to_coarse_dirt:
```
This needs forceInclude: true or it wont be added to the existing factories

Peter5930 replied:
Done.

Peter5930 committed:
Update config.yml

Added forceInclude: true to dirt_to_coarse_dirt recipe in bio lab.

Peter5930 committed:
Update config.yml

Added fence gate recipe to the carpentry factory and standardises the carpentry recipes to a 2x output multiplier.  Since the vanilla doors recipe now yields 3 doors instead of 1 for 6 planks, the carpentry factory was twice as expensive as using a crafting table to make doors.

Peter5930 committed:
Update config.yml

Added recipes to the carpentry factory for turning logs and logs_2 into chests with no multiplier as a convenience to save on repetitive strain injuries.

TealNerd commented:
Why did you cut output for fences and ladders?

TealNerd commented:
@Peter5930

Peter5930 committed:
Update config.yml

Added end_gateway to list of unenforceable blocks to prevent acid blocking of otherwise indestructible blocks.

Peter5930 commented:
@TealNerd I rebalanced prices to be a consistent approximately x2 multiplier across the board instead of arbitrarily having some recipes output at x4, some at x2 and some being more expensive than vanilla crafting (like doors, which mojang changed to yield 3 doors per 6 planks/ingots, which threw the factory cost from 2.0 way off).  Also someone acid blocked one of the end gateways so I added them to the unreinforceables list.

mike0war commented:
There is an issue in the Bio Lab recipes. On converting Birch to oar, A stack of 64 burns twice and produced 64 correctly, 2 stacks of 64 burns four times correctly but only produces 1 1/2 stacks. 3 stacks of 64 burns 6 times correctly but again only produces 1 1/2 stacks. If you remove the 1/2 stacks as produced one by one you will get the full resultant of 3 stacks.
